src/FRC.NetworkTables/TcpSockets/TcpConnector.cs

```diff
@@ -26,7 +26,10 @@ private static int ResolveHostName(string hostName, out IPAddress[] addr)
                     // Sockets don't all support IPV6
                     if (ipAddress.AddressFamily == AddressFamily.InterNetwork)
                     {
-                        addresses.Add(ipAddress);
+                        if (!addresses.Contains(ipAddress))
+                        {
+                            addresses.Add(ipAddress);
+                        }
```

Comment:
Yes I did run into an issue where the same IP address got added twice and a bunch of SocketException's started showing up. It would seem sensible to make this change since it has zero side effects and can potentially prevent a problem...